expression/builtin_info.go

```diff
@@ -245,6 +248,28 @@ func (b *builtinVersionSig) eval(_ []types.Datum) (d types.Datum, err error) {
 	return d, nil
 }
 
+type tidbVersionFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *tidbVersionFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
+	err := errors.Trace(c.verifyArgs(args))
+	bt := &builtinTiDBVersionSig{newBaseBuiltinFunc(args, ctx)}
+	bt.deterministic = true
+	return bt.setSelf(bt), errors.Trace(err)
+}
+
+type builtinTiDBVersionSig struct {
+	baseBuiltinFunc
```

Comment:
currently we are refactoring our expression evaluation framework, it's better to use the new expression evaluation mechanism: replace `baseBuiltinFunc` with `baseStringBuiltinFunc` and implements `evalString()` instead of `eval()`